bucket/mini-sftp-server.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "2.25",
+    "description": "A simple mini sftp server with GUI controls",
+    "homepage": "http://www.coreftp.com/server/",
+    "license": "Freeware",
```

Comment:
Source?

Reply:
> Source?

Are you asking about the source for the license details? If yes - it is mentioned on the same page.

![image](https://user-images.githubusercontent.com/1783193/87969511-f3c1f280-caca-11ea-8d44-d4639b9e4dc8.png)

If you are asking about the source code - there is none available.